controllers/controller.js

```diff
@@ -3,7 +3,8 @@ const {
     selectArticleById,
     updateVoteCount,
     fetchUsers,
-    fetchArticles
+    fetchArticles,
+    fetchArticleComments
```

Comment:
I think it would make sense for your error handlers to be in their own designated directory. That way we can separate our file structure and make our app easier to navigate.